pkg/gitops/fileprocessor/gotemplates.go

```diff
@@ -0,0 +1,55 @@
+package fileprocessor
+
+import (
+	"bytes"
+	"github.com/pkg/errors"
+	api "github.com/weaveworks/eksctl/pkg/apis/eksctl.io/v1alpha5"
+	"path/filepath"
+	"strings"
+	"text/template"
+)
+
+const (
+	templateExtension = ".templ"
```

Comment:
Über nit: we use `.tmpl` in Flux and other projects, so might as well be consistent?